docs/docsite/.templates/eol_banner.html

```diff
@@ -0,0 +1,6 @@
+{# Creates a banner at the top of the page for EOL versions. #}
+<div id="banner" class="Admonition caution">
+    <p>
+      You are reading an unmaintained version of the Ansible documentation. Unmaintained Ansible versions can contain unfixed security vulnerabilities (CVE). Please upgrade to a maintained version. See <a href="/ansible/latest/">the latest Ansible documentation</a>.
```

Comment:
this is an existing bug, but I just realized that href will redirect eol core docsite to ansible latest. I don't know of an easy way to fix it so maybe I should just open a separate issue for this? It should be /ansible/latest for package releases, and ansible-core/devel for core releases probably.